modules/lazyload/assets/js/frontend.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    const dataAttribute = 'data-e-bg-lazyload-src';
+    const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
+    const lazyloadBackgroundObserver = new IntersectionObserver( ( entries ) => {
+        entries.forEach( ( entry ) => {
+            if ( entry.isIntersecting ) {
+                let [ lazyloadBackground, element ] = [ entry.target, entry.target ];
+                const lazyloadSelector = lazyloadBackground.getAttribute( 'data-e-bg-lazyload-selector' );
+                if ( lazyloadSelector ) {
+                    lazyloadBackground = lazyloadBackground.querySelector( lazyloadSelector );
+                }
+                lazyloadBackground.style.backgroundImage = `url( ${ element.getAttribute( dataAttribute ) } )`;
+                lazyloadBackgroundObserver.unobserve( element );
+                element.classList.add( 'lazyloaded' );
```

Comment:
```suggestion
                element.classList.add( 'e-lazyloaded' );
```

Reply:
We try to make the html content as smaller as we can.